r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -711,3 +711,19 @@ test_that("am/pm mirror lubridate", {
   )
 
 })
+
+test_that("semester", {
+  test_df <- tibble(
+    month = c(1:12, NA),
+    month_char_pad = ifelse(month < 10, paste0("0", month), month),
+    dates = as.Date(paste0("2021-", month_char_pad, "-15"))
```

Comment:
```suggestion
    month_as_int = c(1:12, NA),
    month_as_char_pad = ifelse(month < 10, paste0("0", month), month),
    dates = as.Date(paste0("2021-", month_char_pad, "-15"))
```

and then also test `month_as_int` and `month_as_str_pad` below too

Reply:
I've added those as failing tests.